.github/workflows/deploy.yml

```diff
@@ -1,24 +1,50 @@
 name: Deploy to PaaS
 
 on:
-  deployment
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'The environment to deploy to (qa, staging or production)'
```

Comment:
We might use different environment names in the future. Maybe:
```
The environment to deploy to, eg: qa, staging, production...
```
Also please remove the quotes